forest/sycamore/tests/test_functions.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd
+import os
+
+from forest.sycamore.survey_config import get_all_interventions_dict
+from forest.sycamore.functions import aggregate_surveys
+from forest.sycamore.survey_config import gen_survey_schedule
+
+# Needed so that relative paths in the below file will work
+os.chdir(os.path.join("forest", "sycamore", "tests"))
+
+
+def test_get_interventions():
+    empty_dict = get_all_interventions_dict("empty_intervention_data.json")
+
+    assert empty_dict == {}
+
+    interventions_dict = get_all_interventions_dict(
+        "sample_intervention_data.json")
+
+    assert type(interventions_dict["ntz4apjf"]) is dict
+
+    assert interventions_dict["ntz4apjf"]["Time_0"] == "2021-12-14"
+
+
+def test_sample_agg_data():
+    sample_agg_data = aggregate_surveys("sample_dir", ["idr8gqdh"])
+
+    assert pd.isnull(sample_agg_data.loc[0, "time_prev"])
+
+    assert all(pd.to_datetime(sample_agg_data['timestamp'], unit='ms') ==
+               sample_agg_data['UTC time'])
```

Comment:
`all()` requires an iterable

Reply:
Hmm. I replaced this with np.mean, which I think doesn't necessarily need one